pkg/odo/cli/component/devfile.go

```diff
@@ -152,3 +153,22 @@ func (do *DeleteOptions) DevfileComponentDelete() error {
 	log.Successf("Successfully deleted component")
 	return nil
 }
+
+func validateURL(url []envinfo.EnvInfoURL) {
```

Comment:
shouldn't this return err or something instead of just outputting `log.Warningf`?

Reply:
I believe that for a function called `validateURL` we should be returning bool, or a list of URLs, or something else other than outputting to `warning`

Reply:
I would prefer not to return an error since it's a warning not an error. 
Why do we want a return value for this function? For test purpose?

Reply:
Yes. Test purposes as well as the fact that the name of the function says "validateURL" which is assuming that you're passing in the URL and it's validating it. It would make sense that after validation it'd return a boolean?

Perhaps you can create a separate function that says "validateURL" that validates one URL. And then for this function you can make it as `validateURLS" which will validate all of them?